app/Repositories/Activity/ActivityRepository.php

```diff
@@ -544,7 +544,7 @@ public function clone(Playlist $playlist, Activity $activity, $token)
             'thumb_url' => $new_thumb_url,
             'subject_id' => $activity->subject_id,
             'education_level_id' => $activity->education_level_id,
-            'shared' => $activity->shared,
+            'shared' => Project::where('id', $playlist->project_id)->value('shared'),
```

Comment:
The "shared" for the project can be get using following code

$playlist->project->shared